conda_forge_tick/lazy_json_backends.py

```diff
@@ -308,6 +308,14 @@ def hgetall(self, name: str, hashval: bool = False) -> Dict[str, str]:
         )
 
 
+def _test_and_raise_besides_file_not_exists(e):
+    if isinstance(e, github.UnknownObjectException):
+        return
+    if isinstance(e, github.GithubException) and e.status == 404 and "No object found" in e.data["message"]:
```

Comment:
```suggestion
    if e.status == 404 and "No object found" in e.data["message"]:
```

If we know this is a `GithubException` already we can leave out this check.